horovod/ray/elastic_v2.py

```diff
@@ -205,6 +208,8 @@ class ElasticAdapter(Adapter):
         reset_limit (int): Maximum number of times that the training
             job can scale up or down the number of workers after
             which the job is terminated.
+        cooldown_range (int range): Range of seconds(min, max) a failing
+            host will remain in blacklist.
```

Comment:
same here

Reply:
Done

Reply:
The same example as above would be as useful here, I think.